src/kernels/telemetry/sendKernelTelemetryEvent.ts

```diff
@@ -29,20 +30,20 @@ function incrementStartFailureCount(resource: Resource, eventName: any, properti
 export function sendKernelTelemetryEvent<P extends IEventNamePropertyMapping, E extends keyof P>(
     resource: Resource,
     eventName: E,
-    durationMs?: Record<string, number> | number,
-    properties?: P[E] & { [waitBeforeSending]?: Promise<void> },
-    ex?: Error
+    measures?:

```

Comment:
No changes to the API, just added better type inference

Reply:
Now we'll have type checks for `measures` as well, previously it was a free for all dict

Reply:
That's quite the type 😆